lib/paths-cache.js

```diff
@@ -20,6 +20,15 @@ export default class PathsCache extends EventEmitter {
     this._filePathsByProjectDirectory = new Map()
     this._filePathsByDirectory = new Map()
     this._fileWatchersByDirectory = new Map()
+
+    this.config = {
+      excludeVcsIgnoredPaths: atom.config.get('core.excludeVcsIgnoredPaths'),
```

Comment:
Wont saving the config in the constructor will make it so Atom needs to be restarted every time they change settings in order for the new settings to be used?

Reply:
Yes. That's a trade off that we need to make. I want to move this code to a WebWorker, which means no Atom API is available. We need to get these configs on the main thread and pass it to the worker.

Reply:
Then we should observe the config and restart the worker.

Reply:
This is not a big deal. A lot of packages require reloading for activation of the changes. I will add some messages to config so users become aware of that. 

> Then we should observe the config and restart the worker when it changes. Or at least find a way to change the config on the worker.

We can't do that. The cache is only made once when Atom opens a project.

Reply:
OK I found a way to fix this.

Reply:
Fixed in https://github.com/atom-community/autocomplete-paths/pull/237/commits/c46542b182af24ff56148043d70974ab458c95c4